pypeit/core/save.py

```diff
@@ -129,6 +130,9 @@ def save_1d_spectra_fits(specObjs, header, spectrograph, outfile, helio_dict=Non
                 prihdu.header[key.upper()] = header[key.upper()]
             except KeyError:
                 prihdu.header[key.upper()] = header[key]
+                # TODO FW: commented it out at this moment since the dispname/decker/binning and mjd are not saved in
```

Comment:
Not sure I follow.  
Is this going to break something else?

Reply:
Sorry for  the confusion. Just ignore these messages, I was using it to load old reductions. cleaned it up.